enclave/core/sgx/longjmp.S

```diff
@@ -12,7 +12,7 @@
 //
 //==============================================================================
 
-/* ATTN: WINPORT */
+// Modified from linux/arch/x86/um/setjmp_64.S
```

Comment:
Specify that this is in the Linux kernel.

@mikbras @CodeMonkeyLeet @radhikaj Do we know if using linux kernel code runs us into any licensing issues?

The core is pretty simple, and not really specific to the linux kernel (it occurs in many other open source projects). Its also about 25% modified.

Reply:
Used musl-libc as a base instead so this is no longer relevant